### ISSUE TITLE
Failed to install on Windows (node-gyp rebuild)

### ISSUE THREAD

**Issue by vorou** [viddo/recent-files-fuzzy-finder#5]
Win10, Atom 1.0.3.
Here's the `apm install` output:

```
> apm install recent-files-fuzzy-finder
Installing recent-files-fuzzy-finder to C:\Users\vorou\.atom\packages failed

> runas@2.0.0 install C:\Users\vorou\AppData\Local\Temp\apm-install-dir-11572-6480-1997p37\node_modules\recent-files-fuzzy-finder\node_modules\pathwatcher\node_modules\runas
> node-gyp rebuild


C:\Users\vorou\AppData\Local\Temp\apm-install-dir-11572-6480-1997p37\node_modules\recent-files-fuzzy-finder\node_modules\pathwatcher\node_modules\runas>node "C:\Users\vorou\AppData\Local\atom\app-1.0.3\resources\app\apm\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild

gypnpm ERR! Windows_NT 6.2.9200
npm ERR! argv "C:\\Users\\vorou\\AppData\\Local\\atom\\app-1.0.3\\resources\\app\\apm\\bin\\\\node.exe" "C:\\Users\\vorou\\AppData\\Local\\atom\\app-1.0.3\\resources\\app\\apm\\node_modules\\npm\\bin\\npm-cli.js" "--globalconfig" "C:\\Users\\vorou\\.atom\\.apm\\.apmrc" "--userconfig" "C:\\Users\\vorou\\.atom\\.apmrc" "install" "C:\\Users\\vorou\\AppData\\Local\\Temp\\d-11572-6480-csfc1m\\package.tgz" "--target=0.22.0" "--arch=ia32"
npm ERR! node v0.10.35
npm ERR! npm  v2.5.1
npm ERR! code ELIFECYCLE

npm ERR! runas@2.0.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the runas@2.0.0 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the runas package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls runas
npm ERR! There is likely additional logging output above.
```

**Comment by viddo** [comment]
Hey, sorry for late reply… vacations and and all that.

Unfortunately I'm not a windows user myself, but doing a quick search on the error (`Failed at the runas@2.0.0 install script 'node-gyp rebuild') hints that there are issues with compiling native modules on Windows under some circumstances. See:
- https://discuss.atom.io/search?q=Failed%20at%20the%20runas%402.0.0%20install%20script%20%27node-gyp%20rebuild%27
- https://github.com/atom/atom/issues/8279

**Comment by viddo** [comment]
Should not be an issue longer, especially after [last fix released in v0.2.3](https://github.com/viddo/recent-files-fuzzy-finder/commit/53a6b4f55544f9fea0c9fc5cdb9ca7706f74a2da)
